.github/workflows/build.yml

```diff
@@ -140,8 +140,9 @@ jobs:
           apt-get update -y && 
           DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get -y install tzdata &&
           apt-get install -y automake libtool make gcc pkg-config bison git &&
-          git clone https://github.com/VirusTotal/yara &&
+          git clone https://github.com/BitsOfBinary/yara &&
```

Comment:
I guess this was a temporary change for debugging the endianness issue, please revert it to its original state.

Reply:
Yes that's right, I'm only keeping this while I'm debugging + adding changes. I'll revert this before tagging you again for review

Reply:
Now reverted this back